pkg/v1/tkg/client/cluster.go

```diff
@@ -259,15 +259,17 @@ func (c *TkgClient) waitForClusterCreation(regionalClusterClient clusterclient.C
 		return errors.Wrap(err, "unable to create workload cluster client")
 	}
 
-	log.Info("Waiting for addons installation...")
-	if err := c.WaitForAddons(waitForAddonsOptions{
-		regionalClusterClient: regionalClusterClient,
-		workloadClusterClient: workloadClusterClient,
-		clusterName:           options.ClusterName,
-		namespace:             options.TargetNamespace,
-		waitForCNI:            true,
-	}); err != nil {
-		return errors.Wrap(err, "error waiting for addons to get installed")
+	if !isTKGSCluster {
+		log.Info("Waiting for addons installation...")
+		if err := c.WaitForAddons(waitForAddonsOptions{
+			regionalClusterClient: regionalClusterClient,
+			workloadClusterClient: workloadClusterClient,
+			clusterName:           options.ClusterName,
+			namespace:             options.TargetNamespace,
+			waitForCNI:            true,
+		}); err != nil {
+			return errors.Wrap(err, "error waiting for addons to get installed")
+		}
 	}
 
 	log.Info("Waiting for packages to be up and running...")
```

Comment:
Question: Is this applicable for TKGs cluster?

Reply:
@prkalle We don't need to monitor packages for TKGS Cluster for now, I did changes to disable it.